org.gnome.OfficeRunner.json

```diff
@@ -17,13 +17,23 @@
         "--socket=pulseaudio"
     ],
     "modules": [
+        {
+            "name": "gnome-common",
+            "sources": [
+                {
+                    "type": "git",
+                    "url": "https://gitlab.gnome.org/GNOME/gnome-common.git",
+                    "commit": "3f52961bcc8828e33535466d60ca3edcdf5175c6"
+                }
+            ]
+        },
         {
             "name": "office-runner",
             "sources": [
                 {
                     "type": "git",
                     "url": "https://gitlab.gnome.org/GNOME/office-runner.git",
-                    "commit": "1.0.3"
+                    "tag": "1.0.3"
```

Comment:
Why does this need to be changed?

Reply:
1.0.3 is not a commit, that'd be invalid, no? I'm not sure how it worked otherwise.

Reply:
If it worked and now doesn't, then the change that causes that needs to be referenced. (Or fixed if it's a bug)

Reply:
Mark this as resolved when you bump the commit to use to include the appstream changes.